src/components/MDXComponents/MDXLink.tsx

```diff
@@ -19,16 +19,25 @@ export const MDXLink = ({
     href = href.slice(0, href.indexOf('#'));
   }
 
+  // For setting up a link when passed only a hash link
   if (!href.includes('/') && href.startsWith('#')) {
     hash = href;
     href = baseURI.replace(platform, '[platform]');
   }
 
+  const decodedUri = decodeURI(href);
```

Comment:
nit: keep URI uppercase for consistency
```suggestion
  const decodedURI = decodeURI(href);
```